Makefile

```diff
@@ -1,55 +0,0 @@
-# Copyright 2008 Google Inc.
-#
-# This program is free software; you can redistribute it and/or
-# modify it under the terms of the GNU General Public License
-# as published by the Free Software Foundation; either version 2
-# of the License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software
-# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-
-VER_MAJOR = 1
-VER_MINOR = 6
```

Comment:
Let's first fix this in upstream as prerequisite patch to the series (at least to this change). Are you okay to add such a patch or should I do? If you are going to do, choose `1.8` perhaps, or `1.7a`, I don't remember if we have any issues if we add a letter at the end of version. In any case this will effectively mean to split this PR to two.